cli/src/core/parse/packageSpec.go

```diff
@@ -0,0 +1,79 @@
+package parse
+
+import (
+	"cli/core"
+	"cli/core/state"
+
+	"github.com/luno/jettison/errors"
+	"github.com/spf13/cobra"
+)
+
+func getCustomPackages(config *core.Config, customPackagePaths []string) []core.CustomPackage {
```

Comment:
Could we add some comments here to explain ?

Maybe create a separate function to get the intersection between two slices ?

Reply:
I renamed the function and added a comment. I refrain from writing a generic function for this use case, because it won't be that generic.